terraform/modules/tenant-apps/main.tf

```diff
@@ -107,14 +138,14 @@ resource "aws_ssm_parameter" "shared_consumer_ddb" {
 # }
 
 # resource "aws_ssm_parameter" "dedicated_payments_bucket" {
-#   count  = var.enable_payments == true ? 1 : 0
+#   count  = var.enable_consumer == true ? 1 : 0
```

Comment:
Maybe the condition is `enable_payments`?

Reply:
Fixed, thanks!